### PR TITLE
AAP-5413: Created AAP 2.1.3 release notes

### PR DESCRIPTION
**Pull request by sayjadha**
This PR addresses JIRA issue [AAP-5413](https://issues.redhat.com/browse/AAP-5413).

Following are the changes:

- Added platform and hub release notes for AAP 2.1.3

Author: sayjadha

### CODE REVIEW

File: release-notes/master.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ include::topics/controller-420.adoc[leveloffset=+2]
 [[anchor-february-2022-release]]
 == February 2022 release
 
+include::topics/aap-213.adoc[leveloffset=+2]
```
Comment (jbradberry) [inline release-notes/master.adoc:24]: Am I mistaken in thinking that `topics/aap-212.adoc` is going to need to be in here as well?  Or are we planning on waiting on the merge of that PR and then resolving the conflict?

Comment (sayjadha) [reply release-notes/master.adoc:24]: Hi @jbradberry : You're right. The AAP 2.1.2 release notes are being tracked in PR #[552](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/pull/552), and will be available once the review is complete and PR is merged.

**Review by tvo318 — CHANGES_REQUESTED** [review]

File: release-notes/topics/aap-213.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* The setup log now gets updated when it is run as `non-root` on bastion host
+* The users' time zones are now populated correctly in the execution environments
+* Installation of automation hub no longer causes SELinux errors
+* Syncing the execution environments from `registry.redhat.io` on automation hub 2.1 no longer cause `[Too many open files]` errors
+* Enhanced the copy process of execution environments so that they require less space in the ``/tmp` directory
+* The receptor no longer fails in FIPS mode
+* Added the `no_proxy` definition in the inventory file so that deployments no longer use proxy when accessing the Private Automation Hub and thereby run successfully
+* The bundled installer no longer fails while finding registry details
+* Bundle installation no longer fails with `skopeo` file permissions error
+* Installation of Red Hat Single Sign On (SSO) on Private Automation Hub no longer fails
```
Comment (ariordan-redhat) [inline release-notes/topics/aap-213.adoc:25]:
```suggestion
* Installation of Red Hat Single Sign On (SSO) on private automation hub no longer fails
```

File: release-notes/topics/aap-213.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* Bundle installation of Private Automation Hub no longer pushes images with missing `ee_image_base` that leads to installation failure
```
Comment (ariordan-redhat) [inline release-notes/topics/aap-213.adoc:26]:
```suggestion
* Bundle installation of private automationhHub no longer pushes images with missing `ee_image_base` that leads to installation failure
```

File: release-notes/topics/aap-213.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+= Ansible Automation Platform 2.1.3
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements
+* Updated the `Openshift-clients` version to 4.10.x
+* Updated the `pulpcore-selinux` version to 1.3.2
+* Updated the `pulp_installer` version to 3.15.9-3
+* Updated the Receptor version to 1.2.3
+* Execution environment images can now be pulled from the automation hub only
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc]: Made the change.

File: release-notes/topics/aap-213.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+* Task container no longer fails for Controller Operator 2.1.3
+* The setup log now gets updated when it is run as `non-root` on bastion host
+* The users' time zones are now populated correctly in the execution environments
+* Installation of automation hub no longer causes SELinux errors
+* Syncing the execution environments from `registry.redhat.io` on automation hub 2.1 no longer cause `[Too many open files]` errors
+* Enhanced the copy process of execution environments so that they require less space in the ``/tmp` directory
+* The receptor no longer fails in FIPS mode
+* Added the `no_proxy` definition in the inventory file, so that deployments no longer use
+* Updated the deployments to use no proxy when accessing the Private Automation Hub
+* Deployments no longer use proxy to access the Private Automation Hub and thereby run successfully
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc]: Made the change.

File: release-notes/topics/aap-213.adoc [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+* Updated the version to Receptor 1.2.3
+* Added an enhancement such that execution environment images can now be pulled from the automation hub only
+* Modified the base images of execution environments so that the controller backups are executed in the container
+
+.Bug fixes
+* Task container no longer fails for Controller Operator 2.1.3
+* The setup log now gets updated when it is run as `non-root` on bastion host
+* The users' time zones are now populated correctly in the execution environments
+* Installation of automation hub no longer causes SELinux errors
+* Syncing the execution environments from `registry.redhat.io` on automation hub 2.1 no longer cause `[Too many open files]` errors
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:19]: Made the edit.

File: release-notes/topics/aap-213.adoc [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+* Enhanced the copy process of execution environments so that they require less space in the ``/tmp` directory
+* The receptor no longer fails in FIPS mode
+* Added the `no_proxy` definition in the inventory file so that deployments no longer use proxy when accessing the Private Automation Hub and thereby run successfully
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:22]: Made the change.

File: release-notes/topics/aap-213.adoc (hunk 2 above)
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:25]: Made the change.

File: release-notes/topics/aap-213.adoc (hunk 3 above)
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:26]: Made the change.

File: release-notes/topics/aap-213.adoc [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+* Installation of Red Hat Single Sign On (SSO) on Private Automation Hub no longer fails
+* Bundle installation of Private Automation Hub no longer pushes images with missing `ee_image_base` and cause the installation to fail
+* When you run the ``./setup.sh -b` command during an upgrade, the API tokens are now maintained
+* The super user role of users can now be changed by administrators only
+* Improved the tasks for collections deletion so that the collections are now deleted from both the repositories and the disks
+* While using Automation Hub 4.5.0 with central authentication, the hub user interface no longer turns unresponsive when the group permissions are modified
+* Added localization support for *Modules*, *Roles*, *Plugins*, and *Dependencies* counter in container list for both card and list view
+* Updated the documentation link to route to the correct version documentation
+* Resolved intermittent issue while trying to sync community `devsec.hardening` collection on Automation Hub 4.4.0
+* Improved the UI of import log and status overflow pages
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply release-notes/topics/aap-213.adoc:34]: Automation controller UI, added the info.

**Comment by sayjadha** [comment]
Hi @tvo318, @ariordan-redhat: Thank you for the review. Your feedback is incorporated.

**Review by ariordan-redhat — APPROVED** [review]

**Pull request merged**